exercises/I-array-properties/exercise.js

```diff
@@ -7,6 +7,11 @@
 
 function isEmpty(arr) {
   return; // complete this statement
+  if (arr.length === 0){
```

Comment:
This code is correct but you forgot to delete the line above that returns from the function as soon as it gets in to it so it will never be ran